config/helm/ciliumnetworkpolicy_capa-controller-manager.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/component: cluster-api
```

Comment:
is `cluster-api` the right name here? I would've expected cluster-api-provider-aws

Reply:
It's consistent with what we have